docs/modules/plugins/partials/plugin-web-app-steps.adoc

```diff
@@ -2,6 +2,24 @@
 :popup-alert: https://www.w3schools.com/js/js_popup.asp
 :close-spec: https://w3c.github.io/webdriver/webdriver-spec.html#close-window[close]
 
+=== Page steps
+
+==== Go to relative URL
+
+A _relative URL_ - points to a file within a web site (like `about.html` or `/products`)
+
+Actions performed at this step:
+
+* Builds the absolute URL by concatenating the base URL and the relative URL;
+* Loads the page with the absolute URL.
+
+[source,gherkin]
+----
+When I go to relative URL `$relativeURL`
+----
+
+* `relativeURL` - A string value of the relative URL
+
```

Comment:
also please add example demoing step usage

Reply:
done

Reply:
still looking for examples

Reply:
added